specs/ceph_pool_usage_data.adoc

```diff
@@ -0,0 +1,175 @@
+= Gather and populate ceph pool usage data to Tendrl
+
+The ceph integration module in Tendrl synchronizes the cluster data to its
+central store. The cluster data should contain the pool utilization data as well
+and store the same in central store.
+
+Additionally trending graphs as well are required for pool utilization data to
+help administrators to decide on expansion of the cluster.
+
+
+== Problem description
+
+Currently the ceph integration module synchronizes the cluster data to its
+central store. The cluster data should also contain the pool utilization data
+fetched and maintained with cluster so that same can be projected in UI for more
+help to administrator.
+
+In addition to instant pool utilization data maintained with cluster data, the
```

Comment:
I suggest restricting this particular specification only to the bridge implementation and creating a separate spec for the monitoring stack, which @anmolbabu can pick up.